schema/deploy/functions/handle_default_form_change_commit.sql

```diff
@@ -13,6 +13,7 @@ declare
   record_id int;
   json_key text;
   json_data jsonb;
+  table_column_name text;
```

Comment:
I think the custom form change commit handlers might also have this bug

Reply:
Fortunately, those commit handlers act differently. They look for specific fields in the form data and if they can't find it, set the value as null. 
This could also be bad I think, if we add a new field to those forms, we need to update its commit handler too.